packages/itwinui-css/src/input-container/input-container.scss

```diff
@@ -37,3 +37,24 @@
 .iui-input-label {
   @include iui-input-label;
 }
+
+// ----------------------------------------------------------------------------
+
+.iui-input-flex-container {
```

Comment:
Since this file now includes input-flex-container code, you can modify my [changeset](
https://github.com/iTwin/iTwinUI/blob/main/.changeset/mighty-cycles-sparkle.md) to just say "Reorganized source file structure. Should have no changes in the CSS output".

Reply:
Updated